_dashboards/management/multi-data-sources.md

```diff
@@ -128,89 +135,212 @@ data_source.authTypes:
+2. From the dropdown, select the appropriate data source. A list of associated detecotrs appears.
+3. Choose a detector from the list and then select **Actions** and then the appropriate edit option from the dropdown.
+4. Enter the applicable settings and configuration details.  
+
+The following GIF illustrates these steps.
 
 <img src="{{site.url}}{{site.baseurl}}/images/dashboards/ad_mds2.gif" alt="Multiple data sources in Anomaly Detection detector page"/>
 
 ### Security
 
-When the data source feature is enabled, you can navigate to **Security** under the **Management** menu. Using role management as an example, you can view all connected data sources in the navigation bar on the upper right and select a specific data source to view its existing roles. To create a new role, select the **Create role** button, which takes you to a new page. Enter the required information and select **Create** to add the new role to the selected data source. The following GIF illustrates these steps.
+When you have set `data_source.enabled:true`, you can view and manage roles for each connected data source. 
+
+1. Navigate to **Management** > **Security** under the main menu. 
+2. Select **Roles** under the left sidebar menu and then select the {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/database-icon.png" class="inline-icon" alt="database icon"/>{:/} icon on the upper-right menu bar. 
+3. From the dropdown, select the appropriate data source and then select the **Create role** button to add a new role.
+4. Enter the required congiguration information and select the **Create** button to save. 
+
+The following GIF illustrates these steps.
 
 <img src="{{site.url}}{{site.baseurl}}/images/dashboards/security_mds1.gif" alt="Multiple data sources in Security plugin"/>
 
 ### Maps
 
-When the data source feature is enabled, you can navigate to **Maps** under the **OpenSearch Plugins** menu. To edit an existing map, select it from the maps list page, which opens the edit page. On the edit page, you can view all available data sources and the ones currently used in the map. To add a new layer, select **Add layer**, and then select **Documents** from the prompt, which opens a flyout. In the flyout, select the index pattern and geospatial field. Note that the data source name is prefixed to the index pattern name. After selecting **Update**, the new layer is added. Select the {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/database-icon.png" class="inline-icon" alt="database icon"/>{:/} icon to verify that a new data source is now being used in the map. The following GIF illustrates these steps.
+When you have set `data_source.enabled:true`, you can view all available data sources and the ones currently used as layers in a map.
```

Comment:
"you can view all available data sources, including the ones currently used as layers, in a map"?